drv/riscv-plic-server/src/main.rs

```diff
@@ -174,7 +174,6 @@ fn main() -> ! {
     // Zero all interrupt sources ...
```

Comment:
Out of date comment.  Should probably update to reflect the fact you are intentionally not zeroing the plic.